README.md

```diff
@@ -83,7 +87,6 @@ Rscript src/chocolate_data_download.R --url = https://github.com/rfordatascience
 ### Split the data set
 
 Running the script train_test_split.R in the 'src' folder as shown below processes the `chocolate.csv` into a `train_df.csv` and a `test_df.csv` using a 70%-30% split.
-```bash
```

Comment:
this line may have been deleted accidentally

Reply:
good catch!